src/main/java/com/synopsys/integration/alert/channel/jira/util/JiraIssueFormatHelper.java

```diff
@@ -31,13 +32,19 @@
 
 import org.apache.commons.lang3.StringUtils;
 
+import com.synopsys.integration.alert.channel.jira.model.IssueContentModel;
+import com.synopsys.integration.alert.common.channel.MessageSplitter;
+import com.synopsys.integration.alert.common.enumeration.ItemOperation;
 import com.synopsys.integration.alert.common.exception.AlertRuntimeException;
 import com.synopsys.integration.alert.common.message.model.ComponentItem;
 import com.synopsys.integration.alert.common.message.model.ComponentKeys;
 import com.synopsys.integration.alert.common.message.model.LinkableItem;
 
 public class JiraIssueFormatHelper {
     private static final int TITLE_LIMIT = 255;
+    private static final int TEXT_LIMIT = 32767;
```

Comment:
Let's make this smaller than the actual number to account for edge cases. Maybe 25,000 or 30,000.

Reply:
I would agree with 30,000